README.adoc

```diff
@@ -163,6 +163,7 @@ spring.cloud.vault:
     scheme: http
     connection-timeout: 5000
     read-timeout: 15000
+    token: some-application-token
```

Comment:
We did not mention the `token` here because authentication is a very individual configuration aspect.